lib/redis_memo/options.rb

```diff
@@ -20,6 +24,10 @@ def initialize(
     @logger = logger
     @global_cache_key_version = global_cache_key_version
     @expires_in = expires_in
+    @max_connection_attempts = ENV['REDIS_MEMO_MAX_ATTEMPTS_PER_REQUEST']&.to_i || max_connection_attempts
```

Comment:
nit: I think it's somewhat clearer to have those passed in as argument:

```ruby
def initialize(
  max_connection_attempts: ENV['REDIS_MEMO_MAX_ATTEMPTS_PER_REQUEST']&.to_i
)
  @max_connection_attempts = max_connection_attempts
end
```

Reply:
We can't pass in the list of env vars of disabled models as arguments cleanly though. I also did this so that the environment variables would be able to override any option value passed in directly as arguments from the client (though practically it doesn't make much of a difference)

Reply:
Ah that's true! I'm good with the current version then!

Reply:
> the environment variables would be able to override any option value

It's just for the REDIS_MEMO_DISABLE_* env vars though right?

looks like the other ones are only used in the initializer anyway